java8/src/test/java/cucumber/runtime/java8/test/LambdaStepdefs.java

```diff
@@ -17,11 +19,14 @@ public LambdaStepdefs() {
             assertNotSame(this, lastInstance);
             lastInstance = this;
         });
-
         Given("^this data table:$", (DataTable peopleTable) -> {
             List<Person> people = peopleTable.asList(Person.class);
             assertEquals("Hellesøy", people.get(0).last);
         });
+        Integer alreadyHadThisManyCukes = 1;
+        Given("^I have 42 cukes in my belly$", () -> {
```

Comment:
Oh well, so much for that technique then ;) If its not caused by the interface methods being declared accepting Object, I have no immediate ideas for workarounds. Maybe merge the other fix and I'll see if there is a way around it at nearest occation. Should have tested it better :)